core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -72,6 +72,10 @@ oppia.factory('AssetsBackendApiService', [
     };
 
     var _getAudioDownloadUrl = function(explorationId, filename) {
+      if (GLOBALS.DEV_MODE) {
+        return '/assets/test/' + filename;
```

Comment:
Can we have /assets/test/ be part of constants.js perhaps? Or rather -- have an assets dir prefix that's one thing in dev and another thing (the GCS url prefix) in prod? Better to do the if/else check in just one place if possible.